include/rocksdb/perf_context.h

```diff
@@ -169,6 +189,8 @@ struct PerfContext {
   uint64_t env_lock_file_nanos;
   uint64_t env_unlock_file_nanos;
   uint64_t env_new_logger_nanos;
+  PerfContextByLevel* perf_context_by_level;
```

Comment:
We mentioned that we can consider to use std::map to support all levels. What's your thoughts on this approach?